en-us/api/Microsoft_Toolkit_Uwp_UI.md

```diff
@@ -1,18 +1,30 @@
----
-permalink: /en-US/api/Microsoft_Toolkit_Uwp_UI.htm
-title: Microsoft.Toolkit.Uwp.UI API 
-description: API page for Microsoft.Toolkit.Uwp.UI
-keywords: windows, app, toolkit, UWP, API
-layout: api
-search.product: eADQiWindows 10XVcnh
----
-
-
-# Microsoft.Toolkit.Uwp.UI namespace
-
-## Classes
-
-The **Microsoft.Toolkit.Uwp.UI** namespace has the following types:
-
-
-| Class | Description |
| --- | --- |
| [BoolToVisibilityConverter](Microsoft_Toolkit_Uwp_UI_Converters_BoolToVisibilityConverter.htm) | This class converts a boolean value into a Visibility enumeration. |
| [CollectionVisibilityConverter](Microsoft_Toolkit_Uwp_UI_Converters_CollectionVisibilityConverter.htm) | This class converts a collection size to visibility. |
| [ConverterTools](Microsoft_Toolkit_Uwp_UI_Converters_ConverterTools.htm) | Static class used to provide internal tools |
| [StringFormatConverter](Microsoft_Toolkit_Uwp_UI_Converters_StringFormatConverter.htm) | This class provides a binding converter to display formatted strings |
| [StringVisibilityConverter](Microsoft_Toolkit_Uwp_UI_Converters_StringVisibilityConverter.htm) | This class converts a string value into a Visibility value (if the value is null or empty returns a collapsed value). |
| [ImageCache](Microsoft_Toolkit_Uwp_UI_ImageCache.htm) | Provides methods and tools to cache images in a temporary local folder |
| [VisualTreeExtensions](Microsoft_Toolkit_Uwp_UI_VisualTreeExtensions.htm) | Defines a collection of extensions methods for UI. |
\ No newline at end of file
+---
+permalink: /en-US/api/Microsoft_Toolkit_Uwp_UI.htm
+title: Microsoft.Toolkit.Uwp.UI API 
+description: API page for Microsoft.Toolkit.Uwp.UI
+keywords: windows, app, toolkit, UWP, API
+layout: api
+search.product: eADQiWindows 10XVcnh
+---
+
+
+# Microsoft.Toolkit.Uwp.UI namespace
+
+## Classes
+
+The **Microsoft.Toolkit.Uwp.UI** namespace has the following types:
+
+
+| Class | Description |
+| --- | --- |
+| [BoolToVisibilityConverter](Microsoft_Toolkit_Uwp_UI_Converters_BoolToVisibilityConverter.htm) | This class converts a boolean value into a Visibility enumeration. |
+| [CollectionVisibilityConverter](Microsoft_Toolkit_Uwp_UI_Converters_CollectionVisibilityConverter.htm) | This class converts a collection size to visibility. |
+| [ConverterTools](Microsoft_Toolkit_Uwp_UI_Converters_ConverterTools.htm) | Static class used to provide internal tools |
+| [StringFormatConverter](Microsoft_Toolkit_Uwp_UI_Converters_StringFormatConverter.htm) | This class provides a binding converter to display formatted strings |
+| [StringVisibilityConverter](Microsoft_Toolkit_Uwp_UI_Converters_StringVisibilityConverter.htm) | This class converts a string value into a Visibility value (if the value is null or empty returns a collapsed value). |
+| [InMemoryStorage](Microsoft_Toolkit_Uwp_UI_InMemoryStorage.htm) | Provides methods to store instances of any type in memory |

```

Comment:
Everything above this line is already in the md file, why are they showing as new?